### PR TITLE
Ft logout

### PR DESCRIPTION
**Pull request by ishimwebernard**
'#### Description of Task to be completed
>Updated the user model 
>Created a middleware that checks of the token is valid
>Wrote tests for my endpoints
#### What are the relevant Trello story?
https://trello.com/c/TX3nGnzo
#### Screenshots (if appropriate)

### CODE REVIEW

**Comment by sebastien15** [comment]
resolve conflicts with develop

File: src/routes/index.js [hunk 1]
```diff
@@ -1,26 +1,14 @@
+/* eslint-disable linebreak-style */
+import dotenv from 'dotenv';
 import { Router } from 'express';
-// import welcomeRoute from './welcome';
-import Badroute from './badroute';
```
Comment (bertinNDAYISHIMIYE) [inline src/routes/index.js:3]: I think we still need this Badroute in our project.

File: src/routes/index.js [hunk 2]
```diff
@@ -1,26 +1,14 @@
...
-
 router.get('/', welcomeMessage);
-
 router.use(url, api);
 
-// router.use('/auth/', Auth);
-
-// router.use('/', welcomeRoute);
-
-// router.use('/', Badroute);
```
Comment (bertinNDAYISHIMIYE) [inline src/routes/index.js:23]: Why ignoring this route?

**Review by bertinNDAYISHIMIYE — COMMENTED** [review]
Left a comment in index file of routes about deleting bad route handler, you may checkout

File: src/routes/index.js (hunk 1 above)
Comment (ishimwebernard) [reply src/routes/index.js:3]: Oh I wasn't aware of that

**Pull request closed without merging**